tests/hooks/use-content-prediction.test.tsx

```
@@ -0,0 +1,45 @@
+import { act, renderHook } from '@testing-library/react'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)